modules/mod_collection.js

```diff
@@ -0,0 +1,24 @@
+export default class Collection {
+  constructor(collectionName, items) {
+    this.collectionName = collectionName;
+    this.items = items;
+  }
+
+  addBook(title, author) {
+    const newBook = {
+      title,
+      author,
+    };
+    this.items.push(newBook);
+    localStorage.setItem(this.collectionName, JSON.stringify(this.items));
+  }
+
+  removeBook(itemToRemove) {
+    this.items.splice(itemToRemove, 1);
+    localStorage.setItem(this.collectionName, JSON.stringify(this.items));
+  }
```

Comment:
- [ ] It will be great to make this an arrow function. Remember we have to ensure our code follows the new es6 syntax. 
**HINT**
```
addBook = (title, author) => {
yourcode 
}
```